lib/dock_health_api/client.rb

```diff
@@ -6,17 +6,13 @@ class Client
     BUFFER = 120
 
     include Singleton
-    
+
     attr_reader :config
 
     def initialize
       @config = DockHealthApi.config
     end
 
-    def active_client
```

Comment:
This didn't work so am removing it.  I think the issue is before it was acting as a class method, which is how we were calling it, `DockHealthApi::Client.active_client` which failed.  We updated those to `DockHealthApi::Client.instance` in stern